pkg/windows/windows.go

```diff
@@ -400,60 +400,24 @@ func (vm *windows) Reinitialize() error {
 	return nil
 }
 
-func (vm *windows) EnsureRequiredServicesStopped() error {
-	// TODO: In this transitional period of migrating services to WICD's responsibility, WICD must be stopped first to
-	//       ensure it does not start any of these stopped services. This code should be removed as part of
-	//       https://issues.redhat.com/browse/WINC-733
-	svc := &service{name: wicdServiceName}
-	if err := vm.ensureServiceNotRunning(svc); err != nil {
-		return errors.Wrapf(err, "could not stop service %s", wicdServiceName)
-	}
-	for _, svcName := range append(RequiredServices, AzureCloudNodeManagerServiceName) {
-		svc := &service{name: svcName}
-		if err := vm.ensureServiceNotRunning(svc); err != nil {
-			return errors.Wrapf(err, "could not stop service %s", svcName)
-		}
+func (vm *windows) RunWICDCleanup(apiServerURL string, watchNamespace string) error {
+	wicdCleanupCmd := fmt.Sprintf("%s cleanup --api-server %s --sa-ca %s --sa-token %s --namespace %s",
+		wicdPath, apiServerURL, wicdCAFile, wicdTokenFile, watchNamespace)
+	if out, err := vm.Run(wicdCleanupCmd, true); err != nil {
+		vm.log.Info("failed to cleanup node", "command", wicdCleanupCmd, "output", out)
+		return err
 	}
 	return nil
 }
 
-// ensureServicesAreRemoved ensures that all services installed by WMCO are removed from the instance
```

Comment:
This is getting hard to review. You are introducing this function in the previous commit and then removing it in this. I see no point in having such history.

Reply:
Probably the name is confusing, I am just removing this function. The one introduced in previous commit is ensureServiceIsRemoved